internal/pkg/resource/resource.go

```diff
@@ -0,0 +1,66 @@
+package resource
+
+import (
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func NewDeployment(name string, namespace string, replicas int32) *appsv1.Deployment {
+	labels := map[string]string{
+		"app.kubernetes.io/name":     "benthos",
+		"app.kubernetes.io/instance": name,
+	}
+
+	return &appsv1.Deployment{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      name,
+			Namespace: namespace,
+		},
+		Spec: appsv1.DeploymentSpec{
+			Replicas: &replicas,
+			Selector: &metav1.LabelSelector{
+				MatchLabels: labels,
+			},
+			Template: corev1.PodTemplateSpec{
+				ObjectMeta: metav1.ObjectMeta{
+					Labels: labels,
+				},
+				Spec: corev1.PodSpec{
+					Containers: []corev1.Container{{
+						Image:           "jeffail/benthos:4.18",
```

Comment:
can we make this configurable with 4.18 as default value?

Reply:
This was originally going to be part of https://github.com/benthosdev/benthos-captain/issues/3, but I've updated it anyway as it was quite small